plugin/modify-env/modify-env.c

```diff
@@ -48,7 +49,16 @@ void dmtcp_event_hook(DmtcpEvent_t event, DmtcpEventData_t *data)
   switch (event) {
   case DMTCP_EVENT_RESTART:
   { int size = 12288;
-    char *buf = read_dmtcp_env_file("dmtcp_env.txt", size);
+    int retval;
+    char env_file[PATH_MAX];
+
+    retval = dmtcp_get_restart_env("DMTCP_ENV_FILE", env_file, PATH_MAX);
```

Comment:
How about adding two macros or consts for the environment variable and the default file name?
```C
#define DMTCP_ENV_VAR             "DMTCP_ENV_FILE"
#define DMTCP_DEFAULT_ENV_FILE    "dmtcp_env.txt"
```